libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2152,7 +2335,7 @@ bool AP_Mission::distance_to_landing(uint16_t index, float &tot_distance, Locati
     // run through remainder of mission to approximate a distance to landing
     for (uint8_t i=0; i<255; i++) {
         // search until the end of the mission command list
-        for (uint16_t cmd_index = index; cmd_index < (unsigned)_cmd_total; cmd_index++) {
+        for (uint16_t cmd_index = index; cmd_index <= (unsigned)_cmd_total; cmd_index++) {
```

Comment:
This change does not look correct. Should probably update line  2411 to be just  `<` too. Does  not make any functional difference as `get_next_cmd` will return false when passed index of `_cmd_total`

Reply:
Updated in both places